docs/connection-string-guide.md

```diff
@@ -0,0 +1,58 @@
+# Connection string guide for SignalR server SDK

```

Comment:
since we are in a progress of moving docs into docs.microsoft.com repo, could we add this article there directly?

Reply:
this doc talks about connection string https://docs.microsoft.com/en-us/azure/azure-signalr/concept-connection-string

Reply:
I'd like to but there it takes time to publish a doc there. Maybe we could have this merged in advance.